test/v5/kyberTradeLogic.js

```diff
@@ -563,10 +579,8 @@ contract('KyberTradeLogic', function(accounts) {
 
             it("T2T, mask in hint (both ways)", async() => {
                 numMaskedReserves = 2;
-                for (let takerFee = 0; takerFee <= 1000; takerFee += 500) {
-                    takerFeeBps = new BN(takerFee);
-                    for (let platformFee = 0; platformFee <= 1000; platformFee += 500) {
-                        platformFeeBps = new BN(platformFee);
+                for (takerFeeBps of takerFeeArray) {
```

Comment:
I see its repeating many times
isn't it better to add mask type loop here?

Reply:
Looping through the different mask types? The expected result is calculated slightly differently for each